### ISSUE TITLE
Quick save key combo on picozx board

### ISSUE THREAD

**Issue by shiprat-code** [fruit-bat/pico-zxspectrum#197]
Hi 

I’m using a picozx board inside a spectrum+ case . Is there a key combo to take a snapshot save? TIA

**Comment by fruit-bat** [comment]
No, I don't think so... adding key combinations on the spectrum keys seems to end up causing trouble for someone... how about I add an entry to the top level menu?... like press menu then Q

**Comment by shiprat-code** [comment]
That would be perfect, thank you so much .

**Comment by fruit-bat** [comment]
Code is written but I have a bunch of tidying up to do before I can do a build... maybe a few days.

**Comment by shiprat-code** [comment]
Thank you 

very much appreciated. I’m in no rush at all , happy to wait . Thanks again

**Comment by fruit-bat** [comment]
New option should be available in the latest builds

**Comment by shiprat-code** [comment]
Thanks so much . Works perfectly . Now I can use my 48k plus keyboard to write programs and save them easily !
